tests/test_algorithm_template_generation.py

```diff
@@ -46,69 +42,6 @@ def test_algorithm_template_run_permissions(tmp_path, testrun_zpath):
 
     template_path = tmp_path / testrun_zpath
 
-    # Run it twice to ensure all permissions are correctly handled
-    for _ in range(0, 2):
-        _test_script_run(
-            script_path=template_path / "do_test_run.sh",
-        )
-
-        # Check if output is generated (ignore content)
-        output_dir = template_path / "test" / "output"
-        for output in algorithm_template_context["algorithm"]["outputs"]:
-            expected_file = output_dir / output["relative_path"]
-            assert expected_file.exists()
```

Comment:
Why does this not need to be tested anymore? Same question for why the save script no longer needs testing.

Reply:
Ah right, clear!